### PR TITLE
web_widget_x2many_2d_matrix: Allow providing params to cell nodes

### PR DESCRIPTION
**Pull request by ghost**
aimed to solve https://github.com/OCA/web/pull/871#discussion_r177673949

### CODE REVIEW

File: web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js [hunk 1]
```diff
@@ -261,9 +261,14 @@ odoo.define('web_widget_x2many_2d_matrix.X2Many2dMatrixRenderer', function (requ
             }
             // TODO roadmap: here we should collect possible extra params
```
Comment (simahawk) [inline web_widget_x2many_2d_matrix/static/src/js/2d_matrix_renderer.js]: then this comment is not needed anymore => you must update readme

File: web_widget_x2many_2d_matrix/static/src/js/widget_x2many_2d_matrix.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ odoo.define('web_widget_x2many_2d_matrix.widget', function (require) {
             );
             this.field_value = node.field_value || this.field_value;
             // TODO: is this really needed? Holger?
```
Comment (simahawk) [inline web_widget_x2many_2d_matrix/static/src/js/widget_x2many_2d_matrix.js]: is this TODO still needed?

**Review by simahawk — CHANGES_REQUESTED** [review]
thanks!
doc update missing + version upgrade missing

File: web_widget_x2many_2d_matrix/README.rst [hunk 3]
```diff
@@ -133,11 +133,6 @@ Now in our wizard, we can use:
 Known issues / Roadmap
 ======================
 
-* Support extra attributes on each field cell via `field_extra_attrs` param.
```
Comment (simahawk) [inline web_widget_x2many_2d_matrix/README.rst:136]: please update the whole readme. You miss the part "You can pass the following parameters:"

Comment (pedrobaeza) [reply web_widget_x2many_2d_matrix/README.rst:136]: Yeah, please include an example of the usage on the README

**Comment by GSLabIt** [comment]
Ready to merge?

File: web_widget_x2many_2d_matrix/static/src/js/widget_x2many_2d_matrix.js [hunk 4]
```diff
@@ -49,10 +49,10 @@ odoo.define('web_widget_x2many_2d_matrix.widget', function (require) {
                 node.y_axis_clickable || '1'
             );
             this.field_value = node.field_value || this.field_value;
-            // TODO: is this really needed? Holger?
+            this.fields_extra_attrs = {};
             for (var property in node) {
                 if (property.startsWith("field_att_")) {
```
Comment (simahawk) [inline web_widget_x2many_2d_matrix/static/src/js/widget_x2many_2d_matrix.js:54]: ES6 to be fixed

**Review by simahawk — CHANGES_REQUESTED** [review]
@joezsweet not at all. And they guy left.
I'll try to take this over ASAP but if you have time... feel free :wink:

**Pull request closed without merging**